go.mod

```diff
@@ -1,36 +1,99 @@
 module github.com/giantswarm/release-operator/v2
 
-go 1.14
+go 1.17
 
 require (
-	github.com/giantswarm/apiextensions/v2 v2.6.2
-	github.com/giantswarm/apiextensions/v3 v3.38.0
+	github.com/giantswarm/apiextensions-application v0.1.0
+	github.com/giantswarm/config-controller v0.4.1-0.20211119173856-8e0598526059
```

Comment:
Needs https://github.com/giantswarm/config-controller/pull/180 to be released.

Reply:
Should be approved, so merging in order should be fine now.